README.md

```diff
@@ -33,6 +33,8 @@ Inside a fresh virtualenv, `cd` into the root folder of this repository (`xblock
 ```bash
 $ pip install -U pip wheel
 $ pip install -r requirements-test.txt
+$ pip install -r $VENV/src/xblock-sdk/requirements/base.txt
+$ pip install -r $VENV/src/xblock-sdk/requirements/test.txt
```

Comment:
Oh, I used $VENV because that's what is in the docs for another xblock (xblock-chat, I think)